pkg/assetfetcher/http.go

```diff
@@ -161,11 +161,12 @@ func (c *NexposeAssetFetcher) makeRequest(ctx context.Context, wg *sync.WaitGrou
 	}
 	for _, asset := range siteAssetResp.Resources {
 		if !asset.hasBeenScanned() {
-			stater.Count("assetreceived.skipped", 1)
+			stater.Count("assetmissingfield", 1, fmt.Sprintf("site:%s", siteID), "TimeStamp")
```

Comment:
question: Why is this a `missingfield` when the if statement only suggests that the asset has not been scanned?

Reply:
It is a little deceptive, perhaps I can rename it? I wanted to include bad scans under the umbrella of not having an IP address, hostname, ID.